python/tests/simplify.py

```diff
@@ -582,12 +582,17 @@ def __init__(self, ts, sample, ancestors):
         for sample_id in sample:
             self.add_ancestry(0, self.sequence_length, sample_id, sample_id)
         self.edge_buffer = {}
+        self.oldest_ancestor_time = max(ts.tables.nodes.time[u] for u in ancestors)
```

Comment:
``ts.nodes_time[u]`` is the preferred approach now - it's a lot faster. Ditto below.